src/tpm.rs

```diff
@@ -162,17 +163,26 @@ pub fn is_vtpm() -> Option<bool> {
  * is_vtpm helper method
  */
 fn get_tpm_manufacturer() -> Option<String> {
-    let (return_output, _return_code, _file_output) = run(
+    let (return_output, _return_code, _file_output) = match run(
         "getcapability -cap 1a".to_string(),
         EXIT_SUCCESS,
         true,
-        false,
-        String::new(),
-    );
-    let content_result = String::from_utf8(return_output);
-    let content = content_result.unwrap();
+        None,
+    ) {
+        Ok((o, c, f)) => {
+            info!(
+                "Successfully retrieve receive TPM capability information."
+            );
+            (o, c, f)
+        }
+        Err(e) => {
+            // need change the function return type
```

Comment:
Seems like it'd make sense to hold off on merging this until we know what the type should be?

Reply:
What I mean here is to change the return type of the function to Result, but changing those function isn't part of this commit. That is why I put a comment here.

Are we able to merge this PR before change other functions' return type to Result<>?

Reply:
Why don't you add another commit to this PR that accomplishes the signature changes you're suggesting?

Reply:
Ok, I got it.